src/components/Migration.ts

```diff
@@ -67,3 +71,45 @@ async function removeV1(store : Store) : Promise<void> {
     })
 }
 migrations.push({ name: 'removeV1', func: removeV1 })
+
+enum CLIMigrationOption {
+    MIGRATE = 'Yes, migrate',
+    IGNORE = 'No, do not migrate',
+}
+
+async function migrateToCLIDataSource(store : Store) : Promise<void> {
+    const configuration = vscode.workspace.getConfiguration('vsflux')
+    if (configuration.get<string>('datasource') !== null) {
+        // Manually setting the value should skip this migration entirely.
+        return
+    }
+    const { stdout, stderr } = await exec('command -v influx')
+    if (stdout.length === 0 || stderr.length > 0) {
```

Comment:
Why not check the error code instead of stdout/stderr?

Reply:
I was seeing some inconsistent errors related to exit codes, but I don't remember the details. I can't seem to reproduce it.